client/src/components/PublicPages/AuditPlanner/ElectionResultsCard.tsx

```diff
@@ -0,0 +1,295 @@
+  }
+  td {
+    padding: 16px;
+  }
+`
+
+const CandidateContainer = styled.div`
+  display: flex;
+
+  .bp3-button {
+    margin-right: 8px;
+  }
+  .bp3-input-group {
+    flex-grow: 1;
+  }
+`
+
+const VotesContainer = styled.div`
+  display: flex;
+`
+
+const AdditionalInputsRow = styled.div`
+  display: flex;
+  margin-bottom: 16px;
+  margin-left: 16px;
+`
+
+const AdditionalInputContainer = styled.div`
+  align-items: start;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+
+  label > span {
+    color: ${Colors.DARK_GRAY5};
+    display: block;
+    font-size: 20px;
+    font-weight: bold;
+    margin-bottom: 12px;
+  }
+`
+
+const CardActionsRow = styled.div`
+  display: flex;
+`
+
+const CardAction = styled(Button)`
+  &.bp3-button {
+    border-radius: 0;
+    border-top: 1px solid ${Colors.LIGHT_GRAY3};
+    box-shadow: none;
+    font-size: 20px;
+    min-height: 60px;
+    width: 50%;
+  }
+  &.bp3-button:focus {
+    z-index: 1; // Ensures that focus outlines aren't covered
+  }
+  &.bp3-button:first-child {
+    border-bottom-left-radius: 3px;
+  }
+  &.bp3-button:last-child {
+    border-bottom-right-radius: 3px;
+  }
+  &.bp3-button:not(:last-child) {
+    border-right: 1px solid ${Colors.LIGHT_GRAY3};
+  }
+`
+
+interface IProps {
+  editable: boolean
+  electionResults: IElectionResults
+  enableEditing: () => void
+  planAudit: () => void
+  setElectionResults: (electionResults: IElectionResults) => void
+}
+
+const ElectionResultsCard: React.FC<IProps> = ({
+  editable,
+  electionResults,
+  enableEditing,
+  planAudit,
+  setElectionResults,
+}) => {
+  const { candidates, numWinners, totalBallotsCast } = electionResults
+  const { confirm, confirmProps } = useConfirm()
+
+  const setCandidate = (candidateIndex: number, candidate: ICandidate) => {
+    setElectionResults({
+      ...electionResults,
+      candidates: [
+        ...candidates.slice(0, candidateIndex),
+        candidate,
+        ...candidates.slice(candidateIndex + 1),
+      ],
+    })
+  }
+
+  const addCandidate = () => {
+    setElectionResults({
+      ...electionResults,
+      candidates: [...candidates, constructNewCandidate(candidates.length)],
+    })
+  }
+
+  const removeCandidate = (candidateIndex: number) => {
+    setElectionResults({
+      ...electionResults,
+      candidates: [
+        ...candidates.slice(0, candidateIndex),
+        ...candidates.slice(candidateIndex + 1),
+      ],
+    })
+  }
+
+  return (
+    <>
+      <Container data-testid="election-results-card">
+        <InnerContainer>
+          <Heading>Election Results</Heading>
+
+          <CandidatesTable>
+            <thead>
+              <tr>
+                <th>Candidate</th>
+                <th>Votes</th>
+              </tr>
+            </thead>
+            <tbody>
+              {electionResults.candidates.map((candidate, i) => (
+                <tr key={candidate.id}>
+                  <td>
+                    {editable ? (
```

Comment:
Really good point. I went with the latter option, though with an important modification. I used readonly inputs instead of disabled inputs, since the former are better for accessibility. A screen reader will completely ignore disabled inputs but will still read aloud readonly inputs

Regarding form height, curious to hear your thoughts as they relate to inline error messages: https://github.com/votingworks/arlo/pull/1592#discussion_r971136440

Reply:
Awesome, I like the readonly solution! I didn't know about that